python/hail/methods/impex.py

```diff
@@ -453,13 +453,13 @@ def import_bed(path, reference_genome=None):
     # Add the variant annotation ``va.cnvRegion: Boolean`` indicating inclusion in
     # at least one interval of the three-column BED file `file1.bed`:
 
-    # >>> bed = methods.import_bed('data/file1.bed')
+    # >>> bed = hl.import_bed('data/file1.bed')
```

Comment:
Should this be uncommented?

Reply:
yes, but don't want to debug if it's wrong as part of this orthogonal PR